aztec-nargo/compile_then_postprocess.sh

```diff
@@ -11,6 +11,9 @@ dir=$(dirname $0)
 NARGO=${NARGO:-"$dir/../noir/noir-repo/target/release/nargo"}
 TRANSPILER=${TRANSPILER:-"$dir/../avm-transpiler/target/release/avm-transpiler"}
 BB=${BB:-"$dir/../barretenberg/cpp/build/bin/bb"}
+CACHE_DIR=${CACHE_DIR:-"$HOME/.aztec/cache"}
```

Comment:
This is ok but I would prefer the cache folder to be within user’s repo, so it’s easier to get a clean state by removing the cache folder. Either target/cache folder (note that the “target” folder is configurable in nargo) or just “cache” folder.

Keeling it local will make it easier for users if there is a bug in caching behaviour as user can see that there is an obvious cache folder.

If you change this, the starter repos’ .gitignore files have to be updated

Reply:
Yeah, I think I agree. I put them on the `target/cache` folder (to avoid .gitignore shenanigans) finally with a few modifications